charts/incubator/firefox/questions.yaml

```diff
@@ -1,38 +1,47 @@
-# Include{groups}
-portals:
-  open:
-# Include{portalLink}
-questions:
```

Comment:
?

Reply:
between two sets of apps that need portal or not, simply got confused. ill be extra careful and recheck it.

Reply:
no worries :D